apps/pie-docs/src/_11ty/shortcodes/indexPageDisplay.js

```diff
@@ -0,0 +1,73 @@
+const { find } = require('@11ty/eleventy-navigation').navigation;
+const fs = require('fs');
+const path = require('path');
+
+/** IndexPageDisplay Shortcode - automatic page rendering:
+ * Render index items in a flexible grid.
+ * Every main item in our menu can now render an index page. This shortcode will loop on every child menu item and create an
+ * interactive card with a link to that menu item. Items can be excluded if specified. Images must exist for items not excluded.
+ * Mobile images are optional, desktop image will be rendered if mobile image is not provided.
+ * A default image will be rendered if desktop image is not provided and a warning is printed to the server console.
+ * Images must be added:
+ *  - within a folder with the main menu item name, for example 'components/' 'foundations/'
+ *  - image should have the name of the sub-menu in snakecase, for example Checkbox Group is called 'checkbox-group'
```

Comment:
These are good instructions, is it worth adding to a readme somewhere?

Reply:
There is some instructions in readme. I will add a note to look into this component for more information.